include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -569,14 +657,15 @@ private :
         // TODO: meridian case optimization
         if (geometry::math::equals(a312, c0) && meridian_not_crossing_pole)
         {
-            boost::tuple<CT,CT> minmax_elem = boost::minmax(lat1, lat2);
-            if (lat3 >= minmax_elem.template get<0>() &&
-                lat3 <= minmax_elem.template get<1>())
+            std::tuple<CT,CT> minmax_elem = std::minmax(lat1, lat2);
```

Comment:
`std::minmax()` returns `std::pair<>`. There is no need to convert it to `std::tuple<>`.

Reply:
you can use `auto const` (and it will be a pair)